lib/ansible/module_utils/docker/common.py

```diff
@@ -846,6 +846,14 @@ def get_before_after(self):
             after[item['name']] = item['parameter']
         return before, after
 
+    def has_difference_for(self, name):
+        '''
+        Returns a boolean if a difference exists for name
+        '''
+        return any([
```

Comment:
How about removing the `[ ]`? That should be more efficient, since `any()` can iterate over the generated sequence and stop when it finds the first match. With the array, first the whole list is generated, and then `any()` iterates over the list elements until it finds a match.

Reply:
Good catch!